LFGBulletinBoard/Dungeons.lua

```diff
@@ -168,31 +168,31 @@ function GBB.GetDungeonNames()
 			["SCH"] = 	"Scholomance",
 			["LBRS"] = 	"Pic Blackrock",
 			["UBRS"] = 	"Pic Blackrock (10)",
-			["RAMPS"] = "Hellfire Citadel: Ramparts",
-			["BF"] = 	"Hellfire Citadel: Blood Furnace",
-			["SP"] = 	"Coilfang Reservoir: Slave Pens",
-			["UB"] = 	"Coilfang Reservoir: Underbog",
-			["MT"] = 	"Auchindoun: Mana Tombs",
-			["CRYPTS"] = "Auchindoun: Auchenai Crypts",
-			["SETH"] = 	"Auchindoun: Sethekk Halls",
-			["OHB"] = 	"Caverns of Time: Old Hillsbrad",
-			["MECH"] = 	"Tempest Keep: The Mechanar",
-			["BM"] = 	"Caverns of Time: Black Morass",
-			["MGT"] = 	"Magisters' Terrace",
-			["SH"] = 	"Hellfire Citadel: Shattered Halls",
-			["BOT"] = 	"Tempest Keep: Botanica",
-			["SL"] = 	"Auchindoun: Shadow Labyrinth",
-			["SV"] = 	"Coilfang Reservoir: Steamvault",
-			["ARC"] = 	"Tempest Keep: The Arcatraz",
+			["RAMPS"] = "Citadelle des Flammes Infernales: Remparts des Flammes infernales",
+			["BF"] = 	"Citadelle des Flammes Infernales: La Fournaise du sang",
+			["SP"] = 	"Réservoir de Glissecroc : Les enclos aux esclaves",
+			["UB"] = 	"Réservoir de Glissecroc : La Basse-tourbière",
+			["MT"] = 	"Auchindoun: Tombes-mana",
+			["CRYPTS"] = "Auchindoun: Cryptes Auchenaï",
+			["SETH"] = 	"Auchindoun: Les salles des Sethekk",
+			["OHB"] = 	"Grottes du Temps: Contreforts de Hautebrande d'antan",
+			["MECH"] = 	"Donjon de la Tempête: Le Méchanar",
+			["BM"] = 	"Grottes du Temps: Le Noir Marécage",
+			["MGT"] = 	"Terrasse des Magistères",
+			["SH"] = 	"Citadelle des Flammes Infernales: Les Salles brisées",
+			["BOT"] = 	"Donjon de la Tempête: Botanica",
+			["SL"] = 	"Auchindoun: Labyrinthe des ombres",
+			["SV"] = 	"Réservoir de Glissecroc : Le Caveau de la vapeur",
+			["ARC"] = 	"Donjon de la Tempête: L'Arcatraz",
 			["KARA"] = 	"Karazhan",
-			["GL"] = 	"Gruul's Lair",
-			["MAG"] = 	"Hellfire Citadel: Magtheridon's Lair",
-			["SSC"] = 	"Coilfang Reservoir: Serpentshrine Cavern",
-			["EYE"] = 	"The Eye",
+			["GL"] = 	"Repaire de Gruul",
+			["MAG"] = 	"Citadelle des Flammes Infernales: Le repaire de Magtheridon",
+			["SSC"] = 	"Réservoir de Glissecroc : Caverne du sanctuaire du Serpent",
+			["EYE"] = 	"L'Œil du cyclone",
 			["ZA"] = 	"Zul-Aman",
-			["HYJAL"] = "The Battle For Mount Hyjal",
-			["BT"] = 	"Black Temple",
-			["SWP"] = 	"Sunwell Plateau",
+			["HYJAL"] = "Sommet d'Hyjal",
+			["BT"] = 	"Temple noir",
+			["SWP"] = 	"Plateau du Puits de soleil",
 			["ONY"] = 	"Repaire d'Onyxia (40)",
 			["MC"] = 	"Cœur du Magma (40)",
```

Comment:
You are overwriting English names. You need to put all the french names under a new variable

Reply:
Sorry but i don't get it, this fix english names in the dungeonNamesLocales.frFR table, dungeonNamesLocales.deDE / esMX / ruRU doesn't use english names

Reply:
oh you are right, my bad I didn't realize it was already in the french table